### ISSUE TITLE
Right window Plot/Table stays not hidden

### ISSUE THREAD

**Issue by HardyKrause** [OpenSourcePhysics/tracker#44]
Thow I uncheck the right window in the menu it appears immediately after setting manually one tracking point.

Especially if you need full screen size (e.g. on tablets) this behavior is not good. If I say hide the windows, it should be hidden also in case of marking track points.

**Comment by dobrown** [comment]
Yes I see this behavior and agree it is not as it should be. I will fix
that in the next release. Thanks for bringing it to my attention.
